src/node.cc

```diff
@@ -2137,6 +2137,32 @@ void Hrtime(const FunctionCallbackInfo<Value>& args) {
   fields[2] = t % NANOS_PER_SEC;
 }
 
+// CPUUsage exposes libuv's uv_getrusage() this-process resource usage accessor,
+// to access ru_utime (user CPU time used) and ru_stime (system CPU time used),
+// which are uv_timeval_t structs (long tv_sec, long tv_usec) as tuples
+// similar to process.hrtime() values.  Only it's seconds/microseconds instead
+// of hrtime()'s seconds/nanoseconds.
```

Comment:
Have to admit, I'm a bit torn on this:

- on one hand, `rusage()` returns seconds / microseconds, and so I'd kinda like to match that
- on the other hand, `process.hrtime()` returns [seconds, nanoseconds], so would be nice to be consistent

I'm kinda leaning towards your suggestion, returning [seconds, nanoseconds], especially since folks may be using both `hrtime()` and `cpuUsage()` together, to get a "cpu usage %" type value.

If we want to do that, then I think we'd just doc that the resolution is currently at the microsecond level, but the value returned is nanoseconds.